### PR TITLE
Hello-rails backend

### PR DESCRIPTION
**Pull request by diegoh40**
- Create a new Rails API app called 'hello-rails-back-end'.
- Initialize your project with Git.
- Make sure that your project has a Postgres database set up.
- Create a table for storing your messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).
This is the Frontend PR open link https://github.com/diegoh40/hello-rails-react-frontend/pull/1 please check in order to complement the requirements of this project, thanks.

### CODE REVIEW

File: app/controllers/greetings_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class GreetingsController < ApplicationController
+  def index; end
+
+  def show
+    @greeting = Greeting.all.sample
+    render json: @greeting
```
Comment (Davidosky007) [inline app/controllers/greetings_controller.rb:6]: *  I get this error in the console while I run the frontend of your app, Kindly fix this as no random greeting is being rendered. The backend API works fine as it renders random greetings 
image
image

**Review by Davidosky007 — COMMENTED** [review]
Hi @diegoh40 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## Highlights
* No linter errors 👍 
* All necessary links provided 👍 
* Well written PR description 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/greetings_controller.rb (hunk 1 above)
Comment (diegoh40) [reply app/controllers/greetings_controller.rb:6]: the error was corrected and updated in the react frontend, you can see de changes in the pull request side I add the screenshot of my browser, link PR forntend https://github.com/diegoh40/hello-rails-react-frontend/pull/1

image

**Review by m05e5 — APPROVED** [review]
Hi @diegoh40 

You did great. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
CleverScornfulBuffalo-size_restricted

## Highlights
✔️ Descriptive PR 
✔️ your linters are all ok

## Optional suggestions


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**m05e5**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**